WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustomsItem: View {
+    /// Whether the item list is collapsed
+    @State private var isCollapsed: Bool = true
+    @ObservedObject var viewModel: WooShippingCustomsItemViewModel
+    @State private var isShowingHSTarrifInfoWebView = false
+    @State private var isShowingCountries = false
+
+    var body: some View {
+        CollapsibleView(isCollapsed: $isCollapsed,
+                        shouldShowDividers: false,
+                        backgroundColor: .clear,
+                        label: {
+            VStack(alignment: .leading, spacing: Layout.collapsibleViewVerticalSpacing) {
+                HStack {
+                    Text(viewModel.title)
+                        .headlineStyle()
+                    Spacer()
+                    Image(systemName: "exclamationmark.circle")
+                        .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                        .renderedIf(viewModel.informationIsMissing)
+                }
+
+                VStack(alignment: .leading, spacing: Layout.collapsibleViewBottomLabelVerticalSpacing) {
+                    HStack {
+                        Text(viewModel.description)
+                        Spacer()
+                        Text(viewModel.hsTariffNumber)
+                    }
+                    HStack {
+                        Text(viewModel.originCountry.name)
+                        Spacer()
+                        Text(viewModel.weightPerUnit)
+                        Text("•")
+                        Text(viewModel.valuePerUnit)
+                    }
+                }.renderedIf(isCollapsed)
+                    .foregroundColor(.primary)
+                    .subheadlineStyle()
+                    .padding(.trailing, Layout.collapsibleViewBottomContentTrailingPadding)
+            }
+            .padding(.top, Layout.collapsibleViewTopPadding)
+        }, content: {
+            VStack(alignment: .leading, spacing: Layout.collapsibleViewVerticalSpacing) {
```

Comment:
Similar to above, it looks like there's extra space between the labels and their corresponding text field/dropdown menu in this view, compared to the design.

Reply:
Good point, done in 111bf7b4528a63ddf03173f0c8eb56ae7f48c828